invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/components/MultipleOptionsSearchBar.js

```diff
@@ -158,7 +149,6 @@ export class MultipleOptionsSearchBarCmp extends Component {
         onResultSelect={this.onBtnSearchClick}
         onSearchChange={this.handleOnSearchChange}
         resultRenderer={(props) => resultRenderer(props, queryString)}
-        onFocus={this.handleFocus}
```

Comment:
if we remove onFocus, can we still navigate to the searchbar by using tab? (I think it could matter for accessibility as well - screen readers, but correct me if I am wrong)

Reply:
Yes, one can reach the search bar with tab (see my keyboard tests above).
What this handler was doing was to submit the form when the button received focused, which is clearly not a behavior we want (it's probably that people tested the previous change with the mouse only).